core/drivers/crypto/se050/adaptors/apis/sss.c

```diff
@@ -162,7 +162,7 @@ sss_status_t se050_enable_scp03(sss_se05x_session_t *session)
 		if (status != kStatus_SSS_Success)
 			continue;
 
-		if (session->subsystem)
+		if (session && session->subsystem)
```

Comment:
Unrelated change. Please move to a separate commit.

Reply:
I have retested that since `se050_session_init` now calls the i2c stubs whenever `CFG_CORE_SE05X_I2C_TRAMPOLINE` is enabled, so this is no longer necessary but it is still a good defense-in-depth as `se050_enable_scp03` is an exported function.

Reply:
There exists a path for REE to enable SCP:

```
PTA_CMD_ENABLE_SCP03
   -> crypto_se_enable_scp03
     -> se050_enable_scp03
```

Now, on systems without native I2C driver in OP-TEE that solely relies on I2C trampolines, `session` is going to be `NULL` initially until SCP is enabled and working. So attempts to 'close' the previous session will just result in null-pointer exception.

Reply:
> Unrelated change. Please move to a separate commit.

this is not correct. The commits are related - it is not a bug but part of his initial implementation

Reply:
> > Unrelated change. Please move to a separate commit.
> 
> this is not correct. The commits are related - it is not a bug but part of his initial implementation

My mistake. Let's have a single commit.